src/sentry/static/sentry/app/components/search/sources/helpSource.tsx

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import debounce from 'lodash/debounce';
+import dompurify from 'dompurify';
+import {withRouter, WithRouterProps} from 'react-router';
+import {
+  SentryGlobalSearch,
+  standardSDKSlug,
+  Result as SearchResult,
+} from '@sentry-internal/global-search';
+
+import withLatestContext from 'app/utils/withLatestContext';
+import {Organization, Project} from 'app/types';
+import parseHtmlMarks from 'app/utils/parseHtmlMarks';
+
+type MarkedText = ReturnType<typeof parseHtmlMarks>;
+
+type ResultItem = {
+  sourceType: 'help';
```

Comment:
I think the `sourceType` could be an enum that you could reuse here

Reply:
Yeah, it def will be once we type more of omni search :) It's a difficult one to type!